sdk/core/src/services/AnchoringService.ts

```diff
@@ -0,0 +1,73 @@
+import { ELEM_ANCHORED_DID_METHOD, ELEM_DID_METHOD, JOLO_DID_METHOD } from '../_defaultConfig'
+import { RegistryApiService } from '@affinidi/internal-api-clients'
+import { KeysService } from '@affinidi/common'
+
+export class AnchoringService {
```

Comment:
Not sure if this even should be called a service.
The only way it's used is: create an instance, and immediately, in the next line, call `anchorDid`.
Might make sense to just `export const anchorDid(api, encryptedSeed, ...)` here.